src/pages/item-card/style.js

```diff
@@ -20,6 +20,39 @@ export const Sections = styled.div`
     margin: 12px 0;
 `;
 
+export const STable = styled.div`
```

Comment:
I think I had removed this code previously because is not used :x can you validate if this is being used?

Reply:
i think its not used, was put there cause auto-merge